llm_evals_w_crowdlab/llm_evals_w_crowdlab.ipynb

```diff
@@ -0,0 +1,7080 @@
+            "f8be2158142447f19bba5889357abc1c",
+            "9fb5588c5f7d418691fc5ce815d42383",
+            "aa9e5107279d46fca40c992b03102c52",
+            "6dd5334fcc8648ed8a72164899335a13",
+            "8d78f2ed550d41418e7854c384c52463",
+            "1f276ec1e4b84f599200bca5c14a1f61",
+            "403c363ff9ce44cb9686259db0fe31e6",
+            "ec5c3cee541e4144976c3e64aa6e69ce",
+            "55853062fa1c4567b54e8202dc3ea9fa",
+            "5cde2bf3bce545008b811be5afb68262",
+            "c5dad19757724fb8a5f95069dcb282a4",
+            "0dd58e70dc2c483b94aa209848f829f0",
+            "83b32b1b16fc47fabc0a6cdb80d393e9",
+            "fb0aec799c1a40f9a9b148921a6421fa",
+            "016c4f9c04b74ebbbbbc7f911adfcd42",
+            "86f076064f1e4f4bbf8512f63b7ec1d9",
+            "47c35d402c76496396356823e7e5529f",
+            "51e3fabc729d4eb59e4d0409a93ab4bb",
+            "b0c7ecd44ccd490f94e33182f59d9fcb",
+            "dd3a3867381b494aa04e301c66c102d1",
+            "20a5a2d7eb7c4d9bbc04a72188facdd4"
+          ]
+        },
+        "id": "da4114da-51ea-4d5d-aa78-a25156935613",
+        "outputId": "18af3b29-36e3-42b1-eda8-013f0bb114ae"
+      },
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "fd3dcf7c89cf4f4aae7095ec921ad53a",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Creating json from Arrow format:   0%|          | 0/4 [00:00<?, ?ba/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "c5dad19757724fb8a5f95069dcb282a4",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Creating json from Arrow format:   0%|          | 0/3 [00:00<?, ?ba/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/plain": [
+              "11356420"
+            ]
+          },
+          "execution_count": 6,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "dataset[\"human\"].to_json(\"human_judgments.json\")\n",
+        "dataset[\"gpt4_pair\"].to_json(\"gpt4_pair_judgments.json\")\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "9e93803e-9eb7-46bb-940f-2048c255ca54",
+      "metadata": {
+        "id": "9e93803e-9eb7-46bb-940f-2048c255ca54"
+      },
+      "source": [
+        "We'll be using the \"human\" data set, which is where different humans evaluated which LLM most correctly answered the question. This task has the additional (realistic) wrinkle that different LLM pairings have different numbers of graders"
```

Comment:
```suggestion
        "Here we'll be using the \"human\" dataset, in which there are answers to questions from multiple LLMs and different humans (authors, experts) evaluated which LLM most correctly answered each question. Here different examples have varying numbers of human evaluators."
```

Reply:
Also add a sentence about what is author / what is expert, since later you specifically compare them.  That was never clarified in neither the article nor this notebook